v1.25/kind/README.md

```diff
@@ -0,0 +1,51 @@
+# KIND
+
+1. Install KIND
+
+Follow the documentation https://kind.sigs.k8s.io/docs/user/quick-start/#installation
+
+
+```
+curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.15.0/kind-linux-amd64
+chmod +x ./kind
+```
+`
+2. Create a Kubernetes cluster
+
+The Kubernetes version depends on the node image chosen, the list with the supported
+Kubernetes versions and its associated image is published with the release notes:
+
+https://github.com/kubernetes-sigs/kind/releases
+
+```
+./kind create cluster --image kindest/node:v1.25.0
```

Comment:
Let's pin the image for reproducibility?
This flag can also be skipped on v0.15.0 as the current 1.25.0 image is the default image for the tool.

Reply:
let's keep it this way, I think that we just have to demonstrate we are not cheating, I don't think that this instructions are meant to be used by users

Reply:
It's meant to be possible to reproduce, and it's not best practice to use kind node images without pinning them per our own directions.

dropping the image flag would be even simpler.